core/video/encoder.go

```diff
@@ -40,11 +40,20 @@ func init() {
 	}
 }
 
+// Checks to make sure an encoder has been set
```

Comment:
Godoc comments [start with name of the symbol being documented](https://blog.golang.org/godoc-documenting-go-code).
Also given that the function returns more than just an error, then we should describe what the function does as opposed to its current usage. Example:

```go
// Encoder returns the path to either the 'avconv' or 'ffmpeg' executable used
// for video encoding. If neither can be found on PATH then an error is
// returned.
```